### ISSUE TITLE
Export / Import Data

### ISSUE THREAD

**Issue by mikaelacaron** [mikaelacaron/brain-marks#106]
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
Button on settings (or different screen) to export the data to a .csv file that could be saved in Files, or elsewhere.

**Describe alternatives you've considered**
I'm open to any suggestions to save data to a cloud database like AWS (because it already uses AWS Amplify DataStore) or some other service.

**Additional context**
Basically a user could "back up" their data with the csv file in Files, or suggest a way to sync the data to a cloud database. This is so users who delete the app and download it again, they don't lose everything.

**Comment by mikaelacaron** [comment]
duplicate lol
